README.md

```diff
@@ -85,6 +77,7 @@ To change the default auth server edit `server/config/*.json` on your deployed i
 ## Servers
 
 * latest development - https://latest.dev.lcip.org/
+* content - https://latest.dev.lcip.org/
```

Comment:
What's the difference between "latest development" and "content"? They both seem to be the same URL.